### DISCUSSION TITLE
Use Database on different document?

### DISCUSSION THREAD

**Discussion by florian-obradovic** [toeverything/AFFiNE#6780] [Q&A]
Can I reference databases from one doc on another doc (like you can do in notion)?
I'd like to create a doc with all my hardware assets on it (laptops, pcs, servers, switches, etc.)
On another doc I wanna show a filtered view of that assets database. For Example: only laptops.

Is that currently possible?
Thanks a lot.

**Comment by bgervan** [comment]
Currently no. 
I got the info on discord, that the ref feature will come in 0.17, which is probably 2-3 months away yet

**Reply by CodingXCat** [reply]
This is the last thing I need to replace notion, hope it coming out soon.
